Tella/Scenes/CommonConnectionReport/Views/Draft/DraftView.swift

```diff
@@ -0,0 +1,288 @@
+    
+    var draftContentView: some View {
+        GeometryReader { geometry in
+            ScrollView {
+                VStack(alignment: .leading) {
+                    if viewModel.hasMoreServer  {
+                        Text(LocalizableReport.reportsSendTo.localized)
+                            .font(.custom(Styles.Fonts.regularFontName, size: 14))
+                            .foregroundColor(Color.white)
+                        
+                        Button {
+                            DispatchQueue.main.async {
+                                self.menuFrame = geometry.frame(in: CoordinateSpace.global)
+                                shouldShowMenu = true
+                            }
+                            
+                        } label: {
+                            HStack {
+                                Text(viewModel.serverName)
+                                    .font(.custom(Styles.Fonts.regularFontName, size: 14))
+                                    .foregroundColor(Color.white.opacity(0.87))
+                                    .padding()
+                                    .frame(maxWidth: .infinity, alignment: .leading)
+                                
+                                Image("reports.arrow-down")
+                                    .padding()
+                                
+                            }
+                        }.background(Color.white.opacity(0.08))
+                            .cornerRadius(12)
+                        
+                        Spacer()
+                            .frame(height: 55)
+                        
+                    } else {
+                        Spacer()
+                            .frame(height: 5)
+                    }
+                    TextfieldView(fieldContent: $viewModel.title,
+                                  isValid: $viewModel.isValidTitle,
+                                  shouldShowError: $viewModel.shouldShowError,
+                                  fieldType: .text,
+                                  placeholder: LocalizableReport.reportsListTitle.localized,
+                                  shouldShowTitle: true)
+                    
+                    Spacer()
+                        .frame(height: 34)
+                    
+                    UnderlinedTextEditorView(placeholder:  LocalizableReport.reportsListDescription.localized,
+                                             fieldContent: $viewModel.description,
+                                             isValid: $viewModel.isValidDescription,
+                                             shouldShowError: $viewModel.shouldShowError,
+                                             shouldShowTitle: true)
+                    
+                    Spacer()
+                        .frame(height: 24)
+                    
+                    AddFilesToDraftView<T>()
+                        .environmentObject(viewModel)
+                    
```

Comment:
we should remove  `.environmentObject(viewModel) `and pass the `viewmodel` as a parameter